### DISCUSSION TITLE
Log files are full

### DISCUSSION THREAD

**Discussion by kosiodg** [dbeaver/dbeaver#23024] [General]
.log files are filling with certain error, added attachment.
Log files are written to constantly after opening file in SQLEditor and connecting it to a database.
stops only after closing the file.
Otherwise there is no visible problem in the UI, working in the editor, etc.
DBeaver version: 23.3.5
log file: [.bak_1.log](https://github.com/dbeaver/dbeaver/files/14459101/default.bak_1.log)

**Comment by ShadelessFox** [comment]
Thanks for the bug report. Fixed in https://github.com/dbeaver/dbeaver/issues/23004 and will be available in DBeaver 24.0.0.
